rslidar_pointcloud/src/rawdata.cc

```diff
@@ -88,19 +92,57 @@ void RawData::loadConfigFile(ros::NodeHandle node, ros::NodeHandle private_nh)
 
   ROS_INFO_STREAM("[cloud][rawdata] initialize resolution type: " << (dis_resolution_mode_?"1 cm":"0.5 cm") << ", intensity mode: " << intensity_mode_);
 
-  private_nh.param<float>("start_clip_angle_deg", start_clip_angle_, 0.F);
-  private_nh.param<float>("end_clip_angle_deg", end_clip_angle_, 360.F);
+  XmlRpc::XmlRpcValue cut_pairs;
+  if (private_nh.getParam("cut_angles", cut_pairs))
+  {
+    if (cut_pairs.getType() != XmlRpc::XmlRpcValue::TypeArray)
+    {
+      std::string fatal = "Parameter 'cut_angles' is has to be of type array";
+      ROS_FATAL_STREAM(fatal);
+      throw std::invalid_argument(fatal);
```

Comment:
Sadly this is called during `on_init()` of the nodelet, which means it neither crashes the nodelet manager, nor does `ros::shutdown()` work.